service/ec2/retryer_test.go

```diff
@@ -0,0 +1,31 @@
+package ec2
+
+import (
+	"testing"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/awstesting/unit"
+)
+
+func TestCustomRetryer(t *testing.T) {
```

Comment:
Would be good to also have a test case to validate the customization does not replace a custom retryer provided by the user.

Reply:
A table testing here would be a great way to covers these different cases easily.